### PR TITLE
v0.1.0

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Pulling latest merges in to master, prepare for v0.1.0 release.

### CODE REVIEW

**Comment by jklenzing** [comment]
Couple changes for consistency across projects.  Let me know if you see anything missing.

File: setup.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 # Full license can be found in License.md
 # -----------------------------------------------------------------------------
 
-import sys
 import os
 from codecs import open
```
Comment (aburrell) [inline setup.py]: Recommend not overwriting existing function `open`

Comment (jklenzing) [reply setup.py]: Updated.  Since I pulled this format from `pysat`, I've initiated changes there too for consistency.

**Comment by jklenzing** [comment]
Updated codecs usage, ready for review.

**Pull request merged**